include/odp/api/spec/shared_memory.h

```diff
@@ -213,22 +211,23 @@ int odp_shm_capability(odp_shm_capability_t *capa);
  * Reserve a contiguous block of shared memory
  *
  * Reserve a contiguous block of shared memory that fulfills size, alignment
- * and shareability (ODP_SHM_* flags) requirements. In general, a name is
- * optional and does not need to be unique. However, if the block will be
+ * and shareability (ODP_SHM_* flags) requirements. By default (with no flags), the memory
+ * block can be shared between all ODP threads (pthreads) of the calling process, and with
+ * ODP processes that are forked from the calling process _after_ the reserve call.
```

Comment:
Marvell implementation doesn't support forking from a ODP process, so sharing of memory wont be possible in case of two primary processes.

Reply:
```
By default (without flags), the memory
block can be shared between all threads of the ODP instance. Memory address (odp_shm_addr())
shareability depends on application memory model and #ODP_SHM_SINGLE_VA flag usage.
```

Updated text in v2. Idea is that with 0 flags, memory block can be shared within instance, but address sharing depend on  application memory model (thread vs process) and may require SINGLE_VA flag.

Reply:
Yes sounds complete now. As per Marvell implementation, by default all threads will share memory and if a secondary process is spawned, it will also share the memory.

Reply:
All three patches looks good
Reviewed-by: Harman Kalra <hkalra@marvell.com>